sample-dcrlnfaucet.conf

```diff
@@ -23,6 +23,15 @@ bind_addr=:80
 ; use the lets encrypt service to get a certificate for it.
 use_le_https=false
 
+; Use the XRealIP to get the real client IP, but only if a reverse
+; proxy or load balancer is correctly setting the X-Forwarded-For and/or
+; X-Real-Ip headers. (Default is false.)
```

Comment:
OK, I see you did comment that a reverse proxy or load balancer must be "correct" setting these, but I'd really comment that this means trusting the proxy/balancer because clients or non-trusted parties can set these headers to anything.  maybe:

```
; If behind a reverse proxy or load balancer, attempt to get the real client IP address from
; the X-Real-Ip and X-Forwarded-For HTTP request header values. X-Real-Ip is checked
; first since nginx may be configured to set it correctly via the ngx_http_realip_module.
; X-Forwarded-For is parsed if X-Real-Ip is empty.
```